implementations/rust/ockam/ockam_entity/src/entity.rs

```diff
@@ -306,3 +311,159 @@ impl ProfileAuth for Entity {
         }
     }
 }
+
+impl Issuer for Entity {
+    fn get_signing_key(&mut self) -> Result<SigningKeyBytes> {
+        if let Some(profile) = self.default_profile_mut() {
```

Comment:
Same here, I would prefer ok_or way to do that to have less nesting, but wonder what do you think about that